pkgs/development/python-modules/oca-port/default.nix

```diff
@@ -0,0 +1,45 @@
+{
+  buildPythonPackage,
+  click,
+  fetchPypi,
+  GitPython,
```

Comment:
gitpython?

Reply:
It's a dependency: https://github.com/OCA/oca-port/blob/cebdc50f06268500783ec5667d6780e6a549d3a9/pyproject.toml#L18

Reply:
Yes, GitPython is `gitpython` in nixpkgs.
This causes the build to fail.

Reply:
Ah sorry, it was building locally and that's why it was confusing to me. Changed now.